src/components/PopoverProvider/index.js

```diff
@@ -24,6 +25,7 @@ function PopoverContextProvider(props) {
         }
         activePopoverRef.current.close();
         activePopoverRef.current = null;
+        ReportActionComposeFocusManager.focus();
```

Comment:
@jjcoffee 
If you can't call this, we can alternatively call this function here and remove onMouseDown.
In `OptionRowLHN.js`
```js
<PressableWithSecondaryInteraction
    ref={popoverAnchor}
    onPress={(e) => {
        if (e) {
            e.preventDefault();
        }
        props.onSelectRow(optionItem, popoverAnchor);
    }}
    onPressOut={onPressOut={ReportActionComposeFocusManager.focus} //add
    // remove this event
    // onMouseDown={(e) => {
    //     if (!e) {
    //         return;
    //     }
    
    //      e.preventDefault();
    // }}
    onSecondaryInteraction={(e) => {
        showPopover(e);
        // DomUtils.blurActiveElement();    //remove
    }}
```

Reply:
Does this still work to keep focus on the composer when clicking the same chat?

Reply:
Yes.

Reply:
This results in the composer not being focused when switching chats. I still think an early return for anything other than a left click in `onMouseDown` might be the best bet here.

Reply:
My solution is working when switching chats. I think `onMouseDown` is used to prevent loss of focus when clicking the same chat.
But if you want to use it, 
```js
onMouseDown={(e) => {
    if (!e) {
        return;
    }
    ReportActionComposeFocusManager.focus();
    if (e.button !== 2) e.preventDefault();
}}
```

Reply:
Ah my bad, I must have tested incorrectly! Retesting and that seems to work. The downside is long-presses focus on the composer still (which would affect tablet users). Do you have any reason you aren't keen on the early return in `onMouseDown`?

Reply:
The solution provided above addresses this issue. The e.preventDefault() function is not called during secondary interactions.

Reply:
@jjcoffee I have changed my solution.

Reply:
@astrohunter62 Can you push the change so I can test tomorrow? Thanks!

Reply:
Yup.